api/api/accounts/views.py

```diff
@@ -1,22 +1,38 @@
 from django.contrib.auth import get_user_model
-from rest_framework import generics
+from rest_framework import status
+from rest_framework.generics import CreateAPIView, RetrieveUpdateAPIView
 from rest_framework.permissions import IsAdminUser
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from api.roles.models import Roles
+from api.roles.serializers import RolesSerializer
 
 from .permissions import IsOwnerOrAdmin
 from .serializers import UserSerializer
 
 User = get_user_model()
 
 
-class CreateUserView(generics.CreateAPIView):
+class CreateUserView(CreateAPIView):
     """ View to create users """
     permission_classes = (IsAdminUser,)
     queryset = User.objects.all()
     serializer_class = UserSerializer
 
 
-class RetrieveUpdateUserView(generics.RetrieveUpdateAPIView):
+class RetrieveUpdateUserView(RetrieveUpdateAPIView):
     """ View to retrieve and update user """
     permission_classes = (IsOwnerOrAdmin,)
     queryset = User.objects.all()
     serializer_class = UserSerializer
+
+
+class ListUserRoles(APIView):
```

Comment:
AFAIU, this won't return paginated response. Would it?

Can using ListAPIView be used to support pagination here? Also, modify tests accordingly.
Using ListAPIView is also a more rest_frameworky way to do this.